book/website/afterword/subprojects.md

```diff
@@ -6,70 +6,130 @@ These subprojects are often time-bound and are supported by community members wh
 
-- [Liz Hare](https://book.the-turing-way.org/afterword/contributors-record#liz-hare)
+- Chair: [Liz Hare](https://book.the-turing-way.org/afterword/contributors-record#liz-hare)
 - [Anne Lee Steele](https://book.the-turing-way.org/afterword/contributors-record#anne-lee-steele)
+- Patricia Loto
+- Precious Onyewuchi
+
+Find details about this WG's work on their GitHub repository](https://github.com/the-turing-way/accessibility-working-group)
 
-### Book Dash Planning Committee and Working Group Leads
+### Book Dash Planning Committee and Working Group
 
 *Project Information: [Chapter in the Community Handbook](https://book.the-turing-way.org/community-handbook/bookdash)*
 
-- Chair: [Esther Plomp](https://book.the-turing-way.org/afterword/contributors-record#esther-plomp)
-- Secretary: [Arielle Bennett](https://book.the-turing-way.org/afterword/contributors-record#arielle-bennett)
+- Chair: [Arielle Bennett](https://book.the-turing-way.org/afterword/contributors-record#arielle-bennett)
 - Secretary: [Emma Karoune](https://book.the-turing-way.org/afterword/contributors-record#emma-karoune)
-- Member: [Carlos Martinez](https://book.the-turing-way.org/afterword/contributors-record#c-martinez)
-- Member: [Susana Roman Garcia](https://book.the-turing-way.org/afterword/contributors-record#susana465)
+- [Esther Plomp](https://book.the-turing-way.org/afterword/contributors-record#esther-plomp)
+- [Alexandra Araujo Alvarez]()
+- [Carlos Martinez](https://book.the-turing-way.org/afterword/contributors-record#c-martinez)
+
+<!--Add the right link - harmonize two repos-->
+Find details about this WG's work on their [GitHub repository]().
+
+### Community Management Working Group
+
+*Project Information: [Find proposal for this new WG on issue]([https://book.the-turing-way.org/community-handbook/bookdash](https://github.com/the-turing-way/the-turing-way/issues/4033))*
 
+- Anne Lee Steele
+- Léllé Demertzi
+- Malvika Sharan
+- Arielle Bennett
+- Emma Karoune
+- Sara Villa
+
+More information will be shared as soon as this WG is operationalised.
 
 ## Funded Projects 
 
-### Professionalising Data Science Roles - Turing's Skills Policy Award
+### Software Sustainability Institute Fellows
 
-*Project Information: [GitHub repo](https://github.com/alan-turing-institute/professionalising-data-science-roles)*
+- Esther Plomp
+- Arielle Bennett
 
-- [Emma Karoune](https://book.the-turing-way.org/afterword/contributors-record#emma-karoune)
-- [Malvika Sharan](https://book.the-turing-way.org/afterword/contributors-record#malvika-sharan)
-- [Alexandra Araujo Alvarez](https://book.the-turing-way.org/afterword/contributors-record#alexandra-araujo-alvarez)
+### Data Science Without Borders
+
+*Project Information: [GitHub Repository](https://github.com/aphrc-dswb/dswb-open-science-capacity-wg?tab=readme-ov-file#connection-with-the-turing-way-and-other-open-science-communities)*
+
+- Malvika Sharan
+- Precious Onyewuchi
```

Comment:
@Preshh0 - if there's anything we are missing here, please feel free to edit!

Reply:
Should we add the pathway - still in progress, as an added resource? @malvikasharan @aleesteele

Reply:
Alright.